pkg/nuctl/command/import.go

```diff
@@ -112,9 +113,41 @@ func (i *importCommandeer) importFunction(ctx context.Context, functionConfig *f
 			FunctionConfig: *functionConfig,
 		})
 
+	if err != nil && autofix {
+		typedErr := err.(*errors.Error)
+		return i.retryImportWithAutofix(ctx, functionConfig, typedErr)
```

Comment:
@TomerShor I was thinking about the case where function has more than 1 validation error, in that case with current approach (where we just check the error message instead of duplicating function validation on nuctl side), we are able to fix only one error in one iteration. Which means that if we meet more than 1 error, we will fix only the first one.

Reply:
That's correct, but if the first validation error is not something we can fix, we won't change anything and just fail again on the same thing.

I saw your `maxRetries` comment and this can go along with something like - retry until error is not fixable OR maxRetries reached.

Reply:
@TomerShor

> I saw your maxRetries comment and this can go along with something like - retry until error is not fixable OR maxRetries reached.

yeah, that was an idea.